### PR TITLE
Updating Security KBs

### PR DESCRIPTION
**Pull request by chadmiddleton**
I updated the KBs for our new CLI integration, RESTful API, and old pages with new info.

### CODE REVIEW

File: Security/centurylink-cloud-guide-to-vormetric-dsm.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{{
-  "title": "CenturyLink Cloud Guide to Vormetric DSM",
+  "title": "IPS - CenturyLink Cloud Guide to Vormetric DSM",
```
Comment (chrislittle) [inline Security/centurylink-cloud-guide-to-vormetric-dsm.md]: why are you changing the title on my vormetric encryption kb?  this is not an IPS kb

Comment (chadmiddleton) [inline Security/centurylink-cloud-guide-to-vormetric-dsm.md]: I thought it was. Sorry, I figured everything in that folder would be our product.

File: Security/ipsAnywhere.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+{{{ "title": "IPS - CLI Install & Uninstall",
+        "date": "1-11-2016",
+        "author": "Client-Security",
+        "attachments": [],
+        "contentIsHTML": false,
+        "sticky": false }}}
+
+# IPS-API
```
Comment (jim-phillips) [inline Security/ipsAnywhere.md]: Remove this.  The title will be used as the top header automatically.  Also, `###` is the smallest header number used in kb articles.

File: Security/ipsAnywhere.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+The IPS-API is a RESTful api.
+IPS or Intrusion Prevention Service will require you to have two things:
+
+* A server with Administration credentials.
+* An account on the [CenturyLink Cloud Platform](https://www.ctl.io/)
+
+### Supported Managed Operating Systems
+Current supported operating systems can be found here [Operating System Support](../Security/supported-ips-oses.md)
+
+## Install
```
Comment (jim-phillips) [inline Security/ipsAnywhere.md]: change to `###`

File: Security/ipsAnywhere.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+| **Name**      | **Description**                            |
+|---------------|--------------------------------------------|
+|CLC            |CenturyLink Cloud                           |
+|CTL_DCC        |CenturyLink Dedicated Cloud Compute         |
+|CTL_VPDC       |CenturyLink Virtual Private Data Center     |
+|AWS            |Amazon Web Services                         |
+|AZURE          |Microsoft Azure                             |
+|OTHER          |Default value if none are submitted         |
+
+## Uninstall
```
Comment (jim-phillips) [inline Security/ipsAnywhere.md]: change to `###`

File: Security/supported-ips-oses.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+        "attachments": [],
+        "contentIsHTML": false,
+        "sticky": false }}}
+
+# Operating System Support
+
+Currently a server has to be on the CenturyLink Cloud Platform to utilize the Blueprint and API Support.
+Operating systems with the CLI Support can reside inside or outside the CenturyLink Platform.  We support all major Cloud providers, such as AWS, Azure, DigitalOcean, etc.  We also support internal private clouds, hybrid clouds, etc.
+
+## Linux
```
Comment (jim-phillips) [inline Security/supported-ips-oses.md]: `###`

File: Security/supported-ips-oses.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+### Ubuntu
+* Ubuntu 12 64-bit [CLI Support](../Security/ipsAnywhere.md)
+* Ubuntu 14 64-bit [CLI Support](../Security/ipsAnywhere.md)
+
+### CentOS
+* CentOS 5 64-bit [CLI Support](../Security/ipsAnywhere.md)
+* CentOS 6 64-bit [CLI Support](../Security/ipsAnywhere.md)
+
+## Windows
```
Comment (jim-phillips) [inline Security/supported-ips-oses.md]: `###`

**Comment by jim-phillips** [comment]
@chadmiddleton Please address comments

**Comment by mramella** [comment]
Jim, per your request, I have made the required updates. 

Thanks,

Matt

File: Security/ips-api.md [hunk 7]
```diff
@@ -10,6 +10,9 @@
 The IPS-API is a RESTful api.
 IPS or Intrusion Prevention Service will require you to have a server and account in the CenturyLink Platform.
 
+### Supported Managed Operating Systems
+Current supported operating systems can be found here [Operating System Support](../Security/supported-ips-oses.md)
+
 ## Authentication
```
Comment (jim-phillips) [inline Security/ips-api.md]: `###` is the highest header size.

File: Security/supported-ips-oses.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
+{{{ "title": "IPS - Supported Operating Systems",
+        "date": "1-13-2016",
+        "author": "Client-Security",
+        "attachments": [],
+        "contentIsHTML": false,
+        "sticky": false }}}
+
+# Operating System Support
```
Comment (jim-phillips) [inline Security/supported-ips-oses.md]: Remove this. -- title will automatically show as top header.

**Pull request merged**